activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+*   `travel/travel_to` travel time helpers, now raise on nested calls, 
+     as this can lead to confusing time stubbing.
+       
+     Instead of:
+     
+         travel_to 2.days.from_now do
+           # 2 days from today
+           travel_to 3.days.from_now do
+             # 5 days from today
+           end          
```

Comment:
Ah, right. But this can be written with already today:

```ruby
travel_to 2.days.from_now do
  # 2 days from today.
end

travel_to 5.days.from_now do
  # 5 days from today.
end
```